src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -339,19 +341,38 @@ private void notifyUser(MediaRequestEntity mediaRequestEntity,
             if (notificationTemplateName.equals(NotificationApi.NotificationTemplate.ERROR_PROCESSING_AUDIO.toString())) {
 
                 List<DefendantEntity> defendantList = mediaRequestEntity.getHearing().getCourtCase().getDefendantList();
+
                 List<String> defendantNames = new ArrayList<>();
-                for (DefendantEntity defendant : defendantList) {
-                    defendantNames.add(defendant.getName());
+                String defendants = NO_DEFENDANTS;
+                if (!defendantList.isEmpty()) {
+                    for (DefendantEntity defendant : defendantList) {
+                        defendantNames.add(defendant.getName());
+
+                    }
+
+                    defendants = defendantNames.stream().collect(Collectors.joining(","));
                 }
 
-                String defendants = defendantNames.stream().collect(Collectors.joining(","));
+                DateTimeFormatter formatter = DateTimeFormatter.ofPattern("HH:mm:ss");
+
+                String courthouseName = mediaRequestEntity.getHearing().getCourtCase().getCourthouse().getCourthouseName() != null
+                    ? mediaRequestEntity.getHearing().getCourtCase().getCourthouse().getCourthouseName() : NOT_AVAILABLE;
+
+                String hearingDate = mediaRequestEntity.getHearing().getHearingDate() != null
```

Comment:
I've just checked and we need to format this date as `10th May 2023`

Reply:
Date using DateTimeFormatter formatterLong = DateTimeFormatter.ofLocalizedDate(FormatStyle.LONG); can only provide  10 May 2023

Reply:
Code has been updated, but, require to put utility methods created into correct package to avoid duplication

Reply:
Found the issue, as LocalDate.now() was being set in public HearingEntity aHearingWith(...). This has been updated to pass in LocalDate parameter. Duplicate code removed as no longer required.